examples/index.js

```diff
@@ -94,19 +94,27 @@ import UseViewportChange from './hooks/UseViewportChange'
 import UseResponsiveValue from './hooks/UseResponsiveValue'
 import UseBreakpointChange from './hooks/UseBreakpointChange'
 import UseResponsiveProps from './hooks/UseResponsiveProps'
-import UseResponsiveComponent from './hooks/UseResponsiveComponent'
 
 storiesOf('Hooks', module)
   .add('useViewportChange', () => <UseViewportChange />)
   .add('useResponsiveValue', () => <UseResponsiveValue />)
   .add('useBreakpointChange', () => <UseBreakpointChange />)
   .add('useResponsiveProps', () => <UseResponsiveProps />)
-  .add('useResponsiveComponent', () => <UseResponsiveComponent />)
 
 /**
  * Utilities
  */
-import QueryExample from './utils/Query'
+
+// MakeResponsive
+import MakeResponsive from './utils/MakeResponsive/MakeResponsive'
+
+storiesOf('Utilities|MakeResponsive', module).add(
```

Comment:
I believe that `makeResponsive` has the only behavior, so nesting test examples with "Default behavior" looks odd.

I recommend to structure them as:

```js
storiesOf('Utilities', module)
  .add('query', ...)
  .add('makeResponsive', ...)
```